introduction.md

```diff
@@ -1 +1,5 @@
 Hello. GIT. Apparently they forgot to add the path we're currently working on to the favorites list... so "JavaScript".
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?